<?xml version="1.0" encoding="UTF-8" ?>
<testsuite errors="0" failures="1" hostname="laptop-PC" name="com.xaviar.domain.PermissionTests" tests="1" time="0.439" timestamp="2013-08-05T04:19:43">
  <properties />
  <testcase classname="com.xaviar.domain.PermissionTests" name="testSomething" time="0.179">
    <failure message="junit.framework.AssertionFailedError: Implement me" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: Implement me
	at com.xaviar.domain.PermissionTests.testSomething(PermissionTests.groovy:15)
</failure>
  </testcase>
  <system-out><![CDATA[--Output from testSomething--
]]></system-out>
  <system-err><![CDATA[--Output from testSomething--
]]></system-err>
</testsuite>
